playbooks/tasks/portal-role-task-mongo-check-docker-service.yml

```diff
@@ -40,7 +36,7 @@
   when: not mongo_service_ok
 
 - name: Wait for mongo container initialization before checking status
-  ansible.builtin.shell: "docker logs mongo | grep '\"Waiting for connections\"'"
+  ansible.builtin.shell: 'docker logs mongo | grep ''"Waiting for connections"'''
```

Comment:
this was automatically transformed by prettier in my vscode, I tested this playbook so if there would be any issue it would not pass successfully I guess

Reply:
this command has a when condition which means that your playbook run might not have trigger this.  We have had a number of issues in the past due to specific quotation requirements. Please revert this.

Reply:
```suggestion
  ansible.builtin.shell: "docker logs mongo | grep '\"Waiting for connections\"'"
```